init.go

```diff
@@ -165,6 +165,16 @@ func DefaultDatastoreConfig() Datastore {
 	}
 }
 
+// DefaultServicesConfig returns the default set of configs for external services.
+func DefaultServicesConfig() Services {
+	return Services{
+		StatusServerDomain: "https://db.btfs.io", // TODO: future -> status.btfs.io
```

Comment:
suggest add address to the services too, which link to the public key to avoid dns fake.

Reply:
@JustinGaoF ok, I understand, so far, only escrow and guard have public keys, correct?